### DISCUSSION TITLE
Those, who don't like RMS or FSF, should just make forks and take new names for their new projects.

### DISCUSSION THREAD

**Discussion by AnimusPEXUS** [rms-support-letter/rms-support-letter.github.io#4547] [General]
FSF, GNU, GPLv* - is authored by Stallman and should remain Stallman's. Anybody who going to steal names, to which Stallman and other true FSF/GNU members, ideologically included so many effort - are simply just raiders.

**Comment by 6r1d** [comment]
There are many loops and hoops they mention, but in the end you are right.
